modules/nw-ovn-kubernetes-features.adoc

```diff
@@ -5,17 +5,19 @@
 [id="nw-ovn-kubernetes-purpose_{context}"]
 = OVN-Kubernetes purpose
 
-The OVN-Kubernetes network plugin is an open-source, fully-featured Kubernetes CNI plugin that uses Open Virtual Network (OVN) to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution. The OVN-Kubernetes network plugin:
+The OVN-Kubernetes network plugin is an open-source, fully-featured Kubernetes CNI plugin that uses Open Virtual Network (OVN) to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution. The OVN-Kubernetes network plugin uses the following technologies:
 
-// OVN (Open Virtual Network) is consistent with upstream usage.
+* OVN to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution.
+* Kubernetes network policy support and logs, including ingress and egress rules.
+* The Generic Network Virtualization Encapsulation (Geneve) protocol rather than Virtual Extensible LAN (VXLAN) to create an overlay network between nodes.
 
-* Uses OVN (Open Virtual Network) to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution.
-* Implements Kubernetes network policy support, including ingress and egress rules.
-* Uses the Geneve (Generic Network Virtualization Encapsulation) protocol rather than VXLAN to create an overlay network between nodes.
+The OVN-Kubernetes network plugin supports the following capabilities:
 
-The OVN-Kubernetes network plugin provides the following advantages over OpenShift SDN.
-
-* Full support for IPv6 single-stack and IPv4/IPv6 dual-stack networking on supported platforms
-* Support for hybrid clusters with both Linux and Microsoft Windows workloads
-* Optional IPsec encryption of intra-cluster communications
-* Offload of network data processing from host CPU to compatible network cards and data processing units (DPUs)
+* Hybrid clusters that can run both Linux and Microsoft Windows workloads. This environment is known as _hybrid networking_.
+* Offloading of network data processing from the host central processing unit (CPU) to compatible network cards and data processing units (DPUs). This is known as _hardware offloading_.
+* IPv4 dual-stack networking on supported platforms.
+* IPv6 single-stack networking on a bare-metal platform.
+* IPv6 dual-stack networking on supported platforms except for {vmw-full}, {ibm-power-name}, {ibm-z-name}, and {rh-openstack} clusters.
```

Comment:
v6-primary dual stack is only supported on baremetal and vsphere right now, I think. Maybe openstack too, but I'm unsure of the status there. For now I suspect it will be easier to list the supported platforms than to keep a list of exceptions to the supported platforms.

Reply:
Updated to "IPv6 dual-stack networking for a cluster running on a bare-metal, a VMWare, or an RHOSP platforms."

Reply:
v6 primary is supported since 4.15 with OpenStack.
The docs states it:
`Alternatively, if you want an IPv6 primary dual-stack cluster, edit the install-config.yaml file following the example below:`
https://docs.openshift.com/container-platform/4.15/installing/installing_openstack/installing-openstack-installer-custom.html#install-osp-deploy-dualstack_installing-openstack-installer-custom

Reply:
Thanks, Maysa!